src/Mod/Draft/importSVG.py

```diff
@@ -244,36 +244,46 @@ def getcolor(color):
     (r, g, b, a)
         RGBA float tuple, where each value is between 0.0 and 1.0.
     """
+    if color == "none":
+        FreeCAD.Console.PrintMessage("Color defined to 'none', defaulting to black\n")
+        return (0.0, 0.0, 0.0, 1.0)
     if color[0] == "#":
-        # Color string '#12ab9f'
-        if len(color) == 7:
+        if len(color) == 7 or len(color) == 9: # Color string '#RRGGBB' or '#RRGGBBAA'
             r = float(int(color[1:3], 16) / 255.0)
             g = float(int(color[3:5], 16) / 255.0)
-            b = float(int(color[5:], 16) / 255.0)
-        # Color string '#1af'
-        elif len(color) == 4:
+            b = float(int(color[5:7], 16) / 255.0)
+            a = 1.0
+            if len(color) == 9:
+                a = float(int(color[7:9], 16) / 255.0)
+                FreeCAD.Console.PrintMessage(f"Non standard color format #RRGGBBAA : {color}\n")
+            return (r, g, b, a)
+        if len(color) == 4: # Color string '#RGB'
             # Expand the hex digits
             r = float(int(color[1], 16) * 17 / 255.0)
             g = float(int(color[2], 16) * 17 / 255.0)
             b = float(int(color[3], 16) * 17 / 255.0)
-        return (r, g, b, 0.0)
-    # Color string 'rgb(0.12,0.23,0.3,0.0)'
-    elif color.lower().startswith('rgb('):
+            return (r, g, b, 1.0)
+    if color.lower().startswith('rgb(') or color.lower().startswith('rgba('): # Color string 'rgb[a](0.12,0.23,0.3,0.0)'
         cvalues = color[3:].lstrip('(').rstrip(')').replace('%', '').split(',')
```

Comment:
This does not work properly for 'rgba' strings.
Suggested improvement:
`cvalues = color.lstrip('rgba(').rstrip(')').replace('%', '').split(',')`

Reply:
Sorry for the delay, I was unaware of the PR.

Just a question: I am right in concluding that `'#RGBA'` does not occur and that `'rgba(0.12,0.23,0.3,0.0)'` is a standard format?

Reply:
Good catch